joark/src/test/kotlin/no/nav/helsearbeidsgiver/inntektsmelding/joark/dokument/PdfDokumentTest.kt

```diff
@@ -124,12 +132,20 @@ class PdfDokumentTest {
             )
         }
     }
-
     private fun writePDF(title: String, im: InntektsmeldingDokument) {
         // val file = File(System.getProperty("user.home"), "/Desktop/$title.pdf")
         val file = File.createTempFile(title, ".pdf")
         val writer = FileOutputStream(file)
         writer.write(PdfDokument(im).export())
         println("Lagde PDF $title med filnavn ${file.toPath()}")
     }
+
+    // Hjelpemetode for å gjøre pdf til tekst for testing
+    private fun extractTextFromPdf(pdf: ByteArray): String? {
+        val pdfReader = PDDocument.load(pdf)
+        val pdfStripper = PDFTextStripper()
+        val allTextInDocument = pdfStripper.getText(pdfReader)
+        pdfReader.close()
+        return allTextInDocument
+    }
```

Comment:
Denne brukes ikke?

Reply:
Vi la den til for å teste noe i pdfen, men trengte den ikke likevel. 
Men kan være nyttig for å teste fremtige endringer i pdfen